### DISCUSSION TITLE
Integration with Obsidian?

### DISCUSSION THREAD

**Discussion by feedhopper** [TechbeeAT/jtxBoard#52] [Ideas]
Hi,

I wondered about the possibility to integrate / connect this with Obsidan.md.
It's the tool of my choice for thinking, writing and organising. The only thing it is missing is proper CalDAV support.

It would be great to use JTX alongsid with obsidian which is based on markdown files and sports a "tasks" plugin. Maybe there is a way to make the two of you meet...

Best and THX

**Comment by patrickunterwegs** [comment]
Hey @feedhopper ! 🙂 
I am not an Obsidian.md user, but I just checked it out. It seems to be a pretty cool tool for markdown. 
Basically providing obsidian notes through CalDAV would be a requirement, that needs to be addressed there. If they offer CalDAV-support with their entries mapped on a VEVENT, the integration could be possible. 

But there are some difficulties: 
- Obsidian follows a technically unstructured approach, when you create tasks, they are actually just markdown tags that are interpreted as checkboxes.
- jtx Board follows a very structured way according to the iCalendar-standard (RFC-5545). Every task is a dedicated entry in the DB and has its own possible parameters. Subtasks are dedicated entries that have a "related-to" field that link to their parents. (Theoretically also child and sibling relations could be possible). You can also have a status, classification, progress and many more attributes on each task. All of that is not possible in the pure markdown approach of obsidian. 

In my opinion it is almost impossible to bring those two approaches together. Some time ago I had a discussion if jtx Board should support markdown-checkboxes. But this would have opened a mix of the two approaches, that would end up two confusion for the users as you could have markdown checkboxes in the text and at the same time structured subtasks linked to a main entry.

So theoretically obsidian could provide all of the notes as a "VJOURNAL" having all the information in the summary and description field which contains the markdown. But: 
- What should jtx Board do with checkboxes in the text - as you're running into the problem mentioned before?
- What should obsidian do with additional information entered in jtx Board (e.g. Url, location, subtasks, ...)? How would subtasks be represented at all as they are another component (VTODO) and not embedded as markdown?

You see, this idea opens a lot of problems... I'm open for discussion, anyway the change would be necessary in obsidian, but I'm afraid this is also not really a direction where they would like to go...
